core/src/main/java/org/elasticsearch/tribe/TribeService.java

```diff
@@ -154,6 +156,12 @@ public static Settings processSettings(Settings settings) {
     public static final Set<String> TRIBE_SETTING_KEYS = Sets.newHashSet(TRIBE_NAME_SETTING.getKey(), ON_CONFLICT_SETTING.getKey(),
             BLOCKS_METADATA_INDICES_SETTING.getKey(), BLOCKS_METADATA_SETTING.getKey(), BLOCKS_READ_INDICES_SETTING.getKey(), BLOCKS_WRITE_INDICES_SETTING.getKey(), BLOCKS_WRITE_SETTING.getKey());
 
+    // these settings should be passed through to each tribe client, if they are not set explicitly
+    private static final List<String> PASS_THROUGH_SETTINGS = Arrays.asList(
+        NetworkService.GLOBAL_NETWORK_HOST_SETTING.getKey(),
+        NetworkService.GLOBAL_NETWORK_BINDHOST_SETTING.getKey(),
+        NetworkService.GLOBAL_NETWORK_PUBLISHHOST_SETTING.getKey()
```

Comment:
I'm doubting here too, though I see the value in a controlled environment. The true long term fix here is testing so we know what we need. Do note that one can set ANY setting on the tribe.* level. This is about defining inheritance.